package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "eth-simple-keyring",
+  "name": "@metamask/eth-simple-keyring",
```

Comment:
A `publishConfig` entry is required when using a namespace. That will need to be added here as well